### PR TITLE
Adds a fake mason for testing

### PR DESCRIPTION
**Pull request by sebastienvas**
This is useful for testing the future changes I am planning to make to Boskos.

### CODE REVIEW

File: boskos/mason/fake-mason/main.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```
Comment (krzyzacy) [inline boskos/mason/fake-mason/main.go]: nit: 2019

Comment (sebastienvas) [reply boskos/mason/fake-mason/main.go]: thanks

**Pull request merged**